client/my-sites/checkout/src/components/wp-checkout.tsx

```diff
@@ -665,7 +668,15 @@ export default function WPCheckout( {
 					<CheckoutFormSubmit
 						validateForm={ validateForm }
 						submitButtonHeader={ <SubmitButtonHeader /> }
-						submitButtonFooter={ <JetpackCheckoutSeals /> }
+						submitButtonFooter={
+							// Temporarily disabling this lint rule until hasCheckoutVersion is removed
+							// eslint-disable-next-line no-nested-ternary
+							isJetpackCheckout() ? (
```

Comment:
This won't work the way you think because `isJetpackCheckout()` actually means "is jetpack _siteless_ checkout". They use the wording "jetpack checkout" all over the code to refer to that, but it doesn't mean "checkout for a jetpack site" 😭 

You need another way to see if this is a jetpack site, and fortunately we already have one hidden inside `JetpackCheckoutSeals`: `hasCartJetpackProductsOnly` should work.